src/js/movies.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @author Jörgen Lindström
+ */
+//console.log('movies.js loaded');
+
+let moviesArray = [];
```

Comment:
I suggest to add the export function directly here for cleaner code :) 
``` export let moviesArray = []; ```

Reply:
True=)